documentation/advanced-features/adding-new-rtypes.md

```diff
@@ -250,8 +251,9 @@ in the source code.
 To run the integration test with the BIND provider:
 
 ```shell
-cd integrationTest              # NOTE: Not needed if already in that subdirectory
-go test -v -verbose -profile BIND
+pushd integrationTest
+go test -v -profile BIND
```

Comment:
The `-verbose` flag is not invalid.  It (along with `-profile`) is parsed by the integration tests.

Reply:
oh, I did not realize - corrected.